build_tools/ci/cpu_comparison/run.py

```diff
@@ -58,23 +79,25 @@ def run(self, config):
         return run_conv_test(config, filename, n_repeats=2)
 
 
-class ConvolutionNHWCQ:
-    def __init__(self):
+class ConvolutionNHWCQ(RunOnTargetDevice):
+    def __init__(self, run_on_target=["npu1_4col"]):
+        super().__init__(run_on_target)
         self.name = "convolution_nhwc_q"
         self.labels = ["Convolution", "ConvolutionNHWCQ"]
 
-    def run(self, config):
+    def _execute(self, config):
```

Comment:
Please note single underscore preceding a member variable/method is an agreed upon convention (a hint) in Python community to indicate that the concerned member variable/method is not supposed to be used outside the class. Therefore we are not supposed to make a direct call to `_execute` for any of the test instances - we are going to be invoking `run` method itself which in turn is going to access this method from within the class.

I'd therefore prefer keeping it this way.